apps/examples/bluetooth/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_BT_UNIT_TEST
+	bool "Enable bt_unit_test"
+	default n
+#	depends on BT_API
```

Comment:
This is not working on protected build. Please add a dependency with !BUILD_PROTECTED

Reply:
Done.